### PR TITLE
Geph updates

### PR DESCRIPTION
**Pull request by samhum**
Merge changes for updated maps and coordinate data for Path-Generator

### CODE REVIEW

File: config-sample.json [hunk 1]
```diff
@@ -191,26 +191,80 @@
...
             "image_path": "maps/TwoMoons.png",
-            "pixel_to_block_ratio": 2.0,
-            "top_left_coordinate_x": -82,
-            "top_left_coordinate_z": -15
+            "pixel_to_block_ratio": 4.0,
+            "top_left_coordinate_x": -70,
+            "top_left_coordinate_z": -50
+        },
+        {
+            "display_name": "TwoMoonsLluna",
```
Comment (jackah2) [inline config-sample.json:207]: Minor change but I'd modify some of the display names to have spaces and what not. Like this could be `Two Moons (Lluna)`

**Review by jackah2 — APPROVED** [review]
These new maps are really cool!!!

File: config-sample.json (hunk 1 above)
Comment (samhum) [reply config-sample.json]: Changed the names of maps that were not consistent with our naming scheme.

**Review by jackah2 — APPROVED** [review]
Looks good!

**Pull request merged**